js/widgets/help.js

```diff
@@ -294,7 +294,7 @@ class HelpWidget {
                             } else if (typeof message[3] === "string") {
                                 // If it is a string, load the macro
                                 // assocuated with this block
-                                const blocksToLoad = getMacroExpansion(message[3], 100, 100);
+                                const blocksToLoad = getMacroExpansion(this.activity,message[3], 100, 100);
```

Comment:
Please add a space after the comma (also on Line 632).

Reply:
@walterbender Done !